kernel/kobject.h

```diff
@@ -23,12 +23,14 @@ struct kobject {
 	kobject_type type;
 	int refcount;
 	int offset;
+	char *intent;
 };
 
 struct kobject *kobject_create_file(struct fs_file *f);
 struct kobject *kobject_create_device(struct device *d);
 struct kobject *kobject_create_graphics(struct graphics *g);
 struct kobject *kobject_create_pipe(struct pipe *p);
+struct kobject *kobject_init();
```

Comment:
If it's private, don't list it here.

Reply:
My intuition is that it should be private, so it has been removed.